chain-impl-mockchain/src/transaction/witness.rs

```diff
@@ -78,7 +78,7 @@ impl WitnessAccountData {
         transaction_id: &TransactionSignDataHash,
         spending_counter: &account::SpendingCounter,
     ) -> Self {
-        let mut v = Vec::with_capacity(65);
+        let mut v = Vec::with_capacity(69);
```

Comment:
I have no idea if this should be `65` or `69`. `tag || block0` || `transaction_id` is  65 bytes so if you add `spending_counter` then it should be 69 so I thought maybe it should be increased